### ISSUE TITLE
Question about the result difference of video paragraph captioning 

### ISSUE THREAD

**Issue by wanghao14** [ttengwang/PDVC#37]
Thanks for the great work!
I notice that in the Table 4 of your paper, PDVC can achieve "B@4 11.80| M 15.93 | C 27.27" in ActivityNet Captions ae-val set, but  it is "B@4 10.18 | M 15.96 | C 20.66" for PDVC with TSN features shown in the [Readme](https://github.com/ttengwang/PDVC#video-paragraph-captioning-1).  I wonder if the two datasets (ActivityNet Captions v.s. ActivityNet Entity) are different that leads to such different results? Looking forward to your reply.

**Comment by wanghao14** [comment]
The two results should all be evaluated with learnt proposals.

**Comment by ttengwang** [comment]
Hi, actually I only report the results with learnt proposals in the readme.

**Comment by wanghao14** [comment]
@ttengwang Yeah, I understood. Thanks for your reply. But I have one more question, I have run "[PDVC with ground-truth proposals](https://github.com/ttengwang/PDVC#video-paragraph-captioning)" on youcook2 with following command:
> config_path=cfgs/yc2_tsn_pdvc_gt.yml
> python train.py --cfg_path ${config_path} --criteria_for_best_ckpt pc --gpu_id 0
But the para_CIDEr is not very good: only 0.26 on validation set after training 30 epoches(Compare with 0.357 of MART). I want to know if this phenomenon is normal or if I have neglected some details.

**Comment by wanghao14** [comment]
It's doesn't matter. Your experimental results are very sufficient. The following result is the output validation on YouCook2 val set of epoch 28:
```
Validation results of iter 38657:
METEOR:0.1049158524914294
Recall:1.0
Precision:1.0
soda_c:0.12777677319530562
para_Bleu_1:0.3928343014791181
para_Bleu_2:0.22551595174128386
para_Bleu_3:0.12577259496428536
para_Bleu_4:0.07105039868345442
para_METEOR:0.15059172210180471
para_ROUGE_L:0.28671837234207104
para_CIDEr:0.2603262337631203
```
It seems that only table2 gives the results of dense captioning on Youcook2 with the predicted proposals, so could you please help me to see if it is normal with groundtruth proposals?

**Comment by ttengwang** [comment]
Yes, the above scores are normal.

**Comment by wanghao14** [comment]
Thanks for your prompt reply! Hope your future research goes well.
